### PR TITLE
Fix Scala deprecation warnings in code and tests.

### PR DESCRIPTION
**Pull request by psuter**

### CODE REVIEW

File: tests/src/actionContainers/NodeJsActionContainerTests.scala [hunk 1]
```diff
@@ -55,7 +55,7 @@ class NodeJsActionContainerTests extends FlatSpec with Matchers {
 
             val argss = List(
                 JsObject("greeting" -> JsString("hi!")),
-                JsObject("numbers" -> JsArray(List(JsNumber(42), JsNumber(1)))))
+                JsObject("numbers" -> JsArray(JsNumber(42), JsNumber(1))))
```
Comment (rabbah) [inline tests/src/actionContainers/NodeJsActionContainerTests.scala]: this is also fixed in a pending pr.

File: tests/src/actionContainers/SwiftActionContainerTests.scala [hunk 2]
```diff
@@ -56,7 +56,7 @@ class SwiftActionContainerTests extends FlatSpec
 
             val argss = List(
                 JsObject("greeting" -> JsString("hi!")),
-                JsObject("numbers" -> JsArray(List(JsNumber(42), JsNumber(1)))))
+                JsObject("numbers" -> JsArray(JsNumber(42), JsNumber(1))))
```
Comment (rabbah) [inline tests/src/actionContainers/SwiftActionContainerTests.scala]: this is also fixed in a pending pr.

File: tests/src/whisk/core/entity/test/SchemaTests.scala [hunk 3]
```diff
@@ -243,10 +243,10 @@ class SchemaTests extends FlatSpec with BeforeAndAfter {
     behavior of "Parameter"
     it should "properly deserialize and reserialize JSON" in {
         val json = Seq[JsValue](
-            JsArray(List(JsObject("key" -> "k".toJson, "value" -> "v".toJson))),
```
Comment (rabbah) [inline tests/src/whisk/core/entity/test/SchemaTests.scala]: these is also fixed in a pending pr... just a heads up (for ordering).

**Comment by rabbah** [comment]
rebased and [merged](https://github.com/openwhisk/openwhisk/commit/9e658f4ee8e2eb15f9cda6c026cc1bdd711ea00d).

**Comment by psuter** [comment]
Merged as 9e658f4ee8e2eb15f9cda6c026cc1bdd711ea00d.

**Pull request closed without merging**